tests/uptane_test.cc

```diff
@@ -422,6 +422,59 @@ TEST(uptane, pet_name_creation) {
   boost::filesystem::remove_all(uptane_test_dir);
 }
 
+/**
+ * \verify{\tst{49}} Check that aktualizr fails on expired methadata
+ */
+TEST(uptane, expires) {
+  Config config;
+  Utils::copyDir("tests/test_data", uptane_test_dir);
+  config.uptane.metadata_path = uptane_test_dir;
+  config.uptane.director_server = tls_server + "/director";
+  config.uptane.repo_server = tls_server + "/repo";
+
+  FSStorage storage(config);
+  HttpFake http(uptane_test_dir);
+  Uptane::TufRepository repo("director", tls_server + "/director", config, storage, http);
+  try {
+    // Check that we don't fail on good metadata.
+    repo.verifyRole(Uptane::Role::Targets(), now, Utils::parseJSONFile("tests/test_data/targets_noupdates.json"));
+  } catch (...) {
+    FAIL();
```

Comment:
Same comment as for #322 - could you use EXPECT_NO_THROW and EXPECT_THROW here?  It is a bit more explicit, and also will check that the type of the exception is correct